docs/ML-Agents-Overview.md

```diff
@@ -350,12 +350,12 @@ training process.
   learn more about adding visual observations to an agent
   [here](Learning-Environment-Design-Agents.md#multiple-visual-observations).
 
-- **Training with Reset Parameter Sampling** - To train agents to be adapt
-  to changes in its environment (i.e., generalization), the agent should be exposed
+- **Training with Environment Parameter Randomization** - To train agents to be adapt
```

Comment:
to be adapt -> to better adapt ?

Reply:
I feel like this whole sentence should be something like "If an agent is exposed to several variations of an environment, it will be more robust (i.e. generalize better) to unseen variations of the environment."